gui.js

```diff
@@ -3467,6 +3467,19 @@ var CGUI = function()
         }
         break;
 
+      case 9: // TAB
+        if (e.shiftKey)
+          setEditMode(mEditMode == EDIT_SEQUENCE ? EDIT_FXTRACK :
```

Comment:
How about EDIT_NONE? My feeling is that if mEditMode is EDIT_NONE, no change should be made.

Reply:
I'm think the current behaviour is quite intuitive. EDIT_NONE is the bpm, rpp and instr presets, so the flow of going from any of these to sequencer felt quite natural.